tee-worker/build.Dockerfile

```diff
@@ -17,7 +18,7 @@
 
 ### Builder Stage
 ##################################################
-FROM integritee/integritee-dev:0.1.13 AS builder
+FROM litentry/litentry-tee-dev:edge AS builder
 LABEL maintainer="zoltan@integritee.network"
```

Comment:
Should we update these while at it for good measure? :grin:

Reply:
Let's visit this image topic later.